auditorium/gulpfile.js

```diff
@@ -13,6 +13,7 @@ var buffer = require('vinyl-buffer')
 var gap = require('gulp-append-prepend')
 var Readable = require('stream').Readable
 var linguasFile = require('linguas-file')
+var CryptoJS = require('crypto-js')
```

Comment:
As this is now exclusively running on the server, can we use the [stdlib `crypto` module](https://nodejs.org/dist/latest-v18.x/docs/api/crypto.html) and not add a dependency?